sharding/src/main/resources/application.conf

```diff
@@ -0,0 +1,83 @@
+akka {
+  loggers = ["akka.event.Logging$DefaultLogger"]
+  extensions = ["com.romix.akka.serialization.kryo.KryoSerializationExtension$"]
+
+  cluster {
+    seed-nodes = [
+      "akka.tcp://cluster@127.0.0.1:2551",
+      "akka.tcp://cluster@127.0.0.1:2552",
+      "akka.tcp://cluster@127.0.0.1:2553"
+    ]
+    metrics.enabled = off
+
+    sharding.role = "shard"
+  }
+
+  actor {
+    provider = cluster
+
+
+    serializers {
+      kryo = "com.romix.akka.serialization.kryo.KryoSerializer"
+    }
+
+    serialization-bindings {
+      "com.lightbend.akka.bench.sharding.BenchEntity$Ping" = kryo
+      "com.lightbend.akka.bench.sharding.BenchEntity$PersistAndPing" = kryo
+      "com.lightbend.akka.bench.sharding.BenchEntity$Pong" = kryo
+      "com.lightbend.akka.bench.sharding.PersistenceHistograms$PrintHistograms$" = kryo
+
+      // persisted events
+      "com.lightbend.akka.bench.sharding.BenchEntity$PingObserved" = kryo
+    }
+
+    # kryo for serialization for lazy reasons
+    kryo  {
+      type = "nograph"
+      idstrategy = "explicit"
+      buffer-size = 4096
+      max-buffer-size = -1
+      use-manifests = false
+      use-unsafe = false
+      post-serialization-transformations = "off"
+      implicit-registration-logging = false
+      kryo-trace = false
+
+      mappings {
+        // messages
+        "com.lightbend.akka.bench.sharding.BenchEntity$Ping" = 20
+        "com.lightbend.akka.bench.sharding.BenchEntity$PersistAndPing" = 21
+        "com.lightbend.akka.bench.sharding.BenchEntity$Pong" = 22
+        "com.lightbend.akka.bench.sharding.PersistenceHistograms$PrintHistograms$" = 23
+
+        // persisted events
+        "com.lightbend.akka.bench.sharding.BenchEntity$PingObserved" = 30
+      }
+
+
+    }
+  }
+
+  remote {
+    netty.tcp {
+      hostname = "127.0.0.1"
+      port = 0 # overriden by app
+    }
+  }
+
+
+  persistence.journal.plugin = "cassandra-journal"
+  persistence.snapshot-store.plugin = "cassandra-snapshot-store"
+
+}
```

Comment:
shall we also make those contact-points available in the root-application.conf? Can do so easily